app/controllers/students_controller.rb

```diff
@@ -28,7 +29,7 @@ def destroy
     
     private
     def load_student
-        @student = Student.find(params[:id])
+        @student = Student.find(params[:id]).decorate
```

Comment:
`decorates_assigned` can be used to automatically decorate Students.
https://github.com/drapergem/draper#when-to-decorate-objects

Reply:
Ah, nice :)